AirCasting/SDSync/Models/CSVMeasurementStream.swift

```diff
@@ -0,0 +1,82 @@
+// Created by Lunar on 06/12/2021.
+//
+
+import Foundation
+
+struct CSVMeasurementStream {
+    let sensorName: String
+    let measurementType: String
+    let measurementShortType: String
+    let unitName: String
+    let unitSymbol: String
+    let thresholdVeryLow: Int
+    let thresholdLow: Int
+    let thresholdMedium: Int
+    let thresholdHigh: Int
+    let thresholdVeryHigh: Int
+    
+    private static let DEVICE_NAME = "AirBeam3"
+    private static let PM_MEASUREMENT_TYPE = "Particulate Matter"
+    private static let PM_MEASUREMENT_SHORT_TYPE = "PM"
+    private static let PM_UNIT_NAME = "microgram per cubic meter"
+    private static let PM_UNIT_SYMBOL = "µg/m³"
+    static let SUPPORTED_STREAMS: [SDCardCSVFileFactory.Header : CSVMeasurementStream] = [
+        SDCardCSVFileFactory.Header.f : CSVMeasurementStream(sensorName: "\(DEVICE_NAME)-F",
```

Comment:
This all is already implemented in MeasurementStream.swift

Reply:
Not quite, those types are slightly different. One has `id`, other do not. Some properties are optional in MeasurementStreeam but aren't in CSVMeasurementStream.
And even if there were basically equal, I still think it's a good idea to distinguish them. Conceptually they represent different things. And our db scheme (`MeasurementStream`) might change while CSV format stays the same (`CSVMeasurementStream`).

Reply:
I agree with @smialko - each use-case should operate on its own data structures even if they are exactly the same as others. We should look at those from a conceptual standpoint - the code is only truly repeated when it is serving the same purpose, not when it has the same structure. 
However I think those default values like "Temperature" and threshold values especially should be placed in some constants or some kind of a template. I can see default threshold changing some day in the future and currently we have those all around the place and it would be difficult to change all at once